### PR TITLE
GP2-2176 VFM backend

### PR DESCRIPTION
**Pull request by bobmeredith**
To do (delete all that do not apply):

 - [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.
 - [ ] Upgraded any vulnerable dependencies.
 - [x] Requirements have been compiled.
 - [ ] Added any new environment variable to vault.
 - [ ] Cleaned up old flags

### CODE REVIEW

File: directory_sso_api_client/user.py [hunk 1]
```diff
@@ -113,14 +109,30 @@ def get_user_lesson_completed(self, sso_session_id, service, lesson_page):
...
+            },
+            authenticator=authenticator,
+        )
 
     def delete_user_lesson_completed(self, sso_session_id, service, lesson):
         authenticator = AuthenticatorNegotiator(sso_session_id=sso_session_id)
         url = self.endpoints['user_lesson_completed']
         return self.delete(url, {'service': service, 'lesson': lesson}, authenticator=authenticator)
+
+    def get_user_questionnaire(self, sso_session_id, service):
```
Comment (depsiatwal) [inline directory_sso_api_client/user.py:128]: Just wondering if questionaire is the best choice for name , oh man I hate naming :-(
conceptually from UI it's a questionaire but  I imagine in the BE it also seems like it becomes part of the user's profile. Hence is questionaire a very soft wording for it.

just my 2p worth i'm not as close to it as you guys so if bob's happy i'm happy too :-)

Comment (rafa-garcia) [reply directory_sso_api_client/user.py:128]: I think the naming is fine, it matches a collection of `Question` objects in SSO and is not linked to a user profile in any way. As @bobmeredith is the 'father' of this endpoint, he might have something to add.

Comment (bobmeredith) [reply directory_sso_api_client/user.py:128]: Seems OK to me - but welcome any suggestion.
I chose 'questionnaire' as it's a bit more general than 'survey' (which is what the VFM thing is variously described as)
I'm sort of expecting this to be used for other things that comprise a list of questions - without really knowing what those things might be.

**Review by bobmeredith — COMMENTED** [review]
LGTM
Thanks for adding tests @rafa-garcia

**Review by rafa-garcia — APPROVED** [review]

**Pull request merged**